### DISCUSSION TITLE
Node constantly loses connection to Bisq

### DISCUSSION THREAD

**Discussion by SepticFrank** [raspiblitz/raspiblitz#4315] [Q&A]
I don't know why I lose connection so often on bisq when setting it to only connect to one node (raspiblitz) over tor. Is it a config setting that is wrong in bitcoind or a raspiblitz setting or something else? I know tor is unreliable but this is very poor connection constantly. Any ideas? 

I wanted to connect the two over LAN instead of tor but I cannot figure this out as well. Anyone?

Thank you

**Comment by rootzoll** [comment]
@Emzy any idea?
